introduction.md

```diff
@@ -0,0 +1,5 @@
+Hello my name is Yuri Reis and I want so much and can't wait for be a excelent programmer.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.